RR_manuscript/supplementary_materials.Rmd

```diff
@@ -26,6 +31,18 @@ library("kableExtra")
 ```
 
 
+```{r}
+#| label: sankey-plot
+#| fig.cap: "Sankey plot illustrating workflow choices from analyst teams."
+#| out.width: "100%"
```

Comment:
Please use Rmd style knitr option, not Quarto style

Reply:
It works in both and all of the code Ive written for the entire project uses the hashpipe code chunks (which are part of RMarkdown).

Reply:
Both are knitr style because it is part of knitr and there are no changes in functionality. Quarto can use both as well (it is backwards compatible for {}). All of our rmd files use knitr, thus it isn't inconsistent. If you really want to press for a change here, then you should change every other Rmd file as well.

Reply:
https://yihui.org/en/2022/01/knitr-news/

(note that quarto wasn't even released until 7 months later)